gsl-config missing in gsl package

### Package description

gsl-config is a part of gsl library. I am trying to install gsl package in R (built from source - not from pointless repository) but I get an error gsl-config not found. I have gsl and gsl-static installed. Still it fails 

trying URL 'https://cran.irsn.fr/src/contrib/gsl_2.1-7.1.tar.gz'
Content type 'application/x-gzip' length 222357 bytes (217 KB)
==================================================
downloaded 217 KB

* installing *source* package ‘gsl’ ...
** package ‘gsl’ successfully unpacked and MD5 sums checked
** using staged installation
**checking for gsl-config... no
configure: error: gsl-config not found, is GSL installed?**
ERROR: configuration failed for package ‘gsl’
* removing ‘/data/data/com.termux/files/usr/local/R-4.1.2/lib/R/library/gsl’

The downloaded source packages are in
        ‘/data/data/com.termux/files/usr/tmp/Rtmp7Mg80N/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages("gsl") :
  installation of package ‘gsl’ had non-zero exit status


### Home page URL

_No response_

### Source code URL

https://github.com/ampl/gsl

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

_No response_

--- a/packages/gsl/build.sh
+++ b/packages/gsl/build.sh
@@ -3,11 +3,13 @@ TERMUX_PKG_DESCRIPTION="GNU Scientific Library (GSL) providing a wide range of m
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.7
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://mirrors.kernel.org/gnu/gsl/gsl-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=efbbf3785da0e53038be7907500628b466152dbc3c173a87de1b5eba2e23602b
 TERMUX_PKG_BREAKS="gsl-dev"
 TERMUX_PKG_REPLACES="gsl-dev"
-TERMUX_PKG_RM_AFTER_INSTALL="bin/ share/man/man1"
+# Do not remove `bin/gsl-config`
+#TERMUX_PKG_RM_AFTER_INSTALL="bin/ share/man/man1"
 
 # Workaround for linker on Android 5 (fixed in Android 6) not supporting RTLD_GLOBAL.
 # See https://github.com/termux/termux-packages/issues/507

